cylc/uiserver/data_store_mgr.py

```diff
@@ -408,3 +389,76 @@ async def entire_workflow_update(self, ids=None):
                             continue
                         new_data[field.name] = {n.id: n for n in value}
                     self.data[w_id] = new_data
+                    successes.add(w_id)
+                else:
+                    self.log.error(
+                        f'Error: communicating with {w_id} - {result}'
+                    )
+
+        return successes
+
+    def _update_contact(
+        self,
+        w_id,
+        contact_data=None,
+        status=None,
+        status_msg=None,
+        pruned=False,
+    ):
+        delta = DELTAS_MAP[ALL_DELTAS]()
+        delta.workflow.time = time.time()
+        flow = delta.workflow.updated
+        flow.id = w_id
+        flow.stamp = f'{w_id}@{delta.workflow.time}'
+        if contact_data:
+            # update with contact file data
+            flow.name = contact_data['name']
+            flow.owner = contact_data['owner']
+            flow.host = contact_data[CFF.HOST]
+            flow.port = int(contact_data[CFF.PORT])
+            # flow.pub_port = int(contact_data[CFF.PUBLISH_PORT])
```

Comment:
Is this needed?

Reply:
No, this attribute is unused, I had commented it out but not removed it, removed now.